template.rs

```diff
@@ -11,6 +11,9 @@ __PROBLEM_DEFAULT_CODE__
 
 // submission codes end
 
+// problem: __PROBLEM_LINK__
+// discuss: __DISCUSS_LINK__
```

Comment:
I'd suggest we keep the informations before the submission codes (closer to the problem description, and make the submission and test case compact)

Reply:
sure, fixed https://github.com/aylei/leetcode-rust/pull/26/commits/e62f13afe8c69b69ca8b2949179c02ff706a7d58